modules/client/src/internal/client/methods.ts

```diff
@@ -625,4 +637,62 @@ export class ClientMethods extends ClientCore implements IClientMethods {
       ),
     );
   }
+
+  public async getPlugins({
+    limit = 10,
+    skip = 0,
+    direction = SortDirection.ASC,
+    sortBy = PluginSortBy.SUBDOMAIN,
```

Comment:
Why having a sortBy parameter if only one field is available?

Reply:
In the future we may have more than one, like, ratings, recomended, most installed... Thats why I left it there